lang/python/python-awesomeversion/Makefile

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (C) 2021 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-awesomeversion
+PKG_VERSION:=21.8.1
+PKG_RELEASE:=$(AUTORELEASE)
+
+PYPI_NAME:=awesomeversion
+PKG_HASH:=a72f3fff12df257e30f5e25a8550fc4da62e7591d5fdb70714f5c96827c37645
+
+PKG_MAINTAINER:=Josef Schlehofer <josef.schlehofer@nic.cz>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE.md
```

Comment:
```suggestion
PKG_LICENSE_FILES:=LICENCE.md
```

Reply:
wow, that's a good eye
any way to automate this check through CI?

or, is there a CI check for this?
pinging a few people on this question (for ideas)   @neheb @hnyman 

i feel like we may have missed a few along the way;

Reply:
@aparcar is the CI person.

Reply:
Any ideas how to implement this? Wondering if this could become part of package/foobar/check? We'd need to unpack the downloaded file and see if the license exists in BUILD_DIR

Reply:
> Any ideas how to implement this? Wondering if this could become part of package/foobar/check? We'd need to unpack the downloaded file and see if the license exists in BUILD_DIR

yes, i think that would be the approach to do it;
so, something like
```
foreach(license_files in $(PKG_LICENSE_FILES)) {
    if (not file.exists(PKG_BUILD_DIR +  license_file)
         fail_build
}
```

Reply:
If you want to go crazy you can also use a tool like [identify](https://github.com/pre-commit/identify#identifying-license-files) or [lc](https://github.com/boyter/lc/) to verify the SPDX ids in `PKG_LICENSE` against the files in `PKG_LICENSE_FILES`.

Reply:
I'm currently stuck on how to figure out `PKG_BUILD_DIR` from the package name. So ideally something like `package/<foobar>/dumpinfo` would print such information. It's possible to use `make -C package/utils/<foobar>/ va.PKG_BUILD_DIR` however I don't have the full path, just the name...